# bed

For BED files

Contents:


* :sleeping_bed: /bed_files/b37/kits/gemini/refseq_nirvana_203.bed
> This is the bed file containing RefSeq exons +/- 0bp, extracted from the Nirvana 2.0.3 Cache GRCH37 RefSeq .gff. This file delineates the regions for coverage analysis in Gemini/Dias. Obsolete.
* :sleeping_bed: /bed_files/b37/kits/gemini/refseq_nirvana_203_100bp_flank.bed
> This is the bed file containing RefSeq exons +/- 100bp, extracted from the Nirvana 2.0.3 Cache GRCH37 RefSeq .gff. This file delineates the regions that are variant called by Gemini/Dias. Obsolete.

* :sleeping_bed: /bed_files/b37/kits/gemini/refseq_nirvana_2.0.10.bed
> This is the bed file containing RefSeq exons +/- 0bp, extracted from the Nirvana 2.0.10 Cache GRCH37 RefSeq .gff.
* :sleeping_bed: /bed_files/b37/kits/gemini/refseq_nirvana_2.0.10_5bp.bed
> This is the bed file containing RefSeq exons +/- 5bp, extracted from the Nirvana 2.0.10 Cache GRCH37 RefSeq .gff.
* :sleeping_bed: /bed_files/b37/kits/gemini/refseq_nirvana_2.0.10_100bp.bed
> This is the bed file containing RefSeq exons +/- 100bp, extracted from the Nirvana 2.0.10 Cache GRCH37 RefSeq .gff.

* :sleeping_bed: /bed_files/b37/kits/gemini/refseq_nirvana_2.0.10_no_PAR_Y.bed
> This is the bed file containing RefSeq exons +/- 0bp, extracted from the Nirvana 2.0.10 Cache GRCH37 RefSeq .gff. Regions in PAR on Y have been removed.
* :sleeping_bed: /bed_files/b37/kits/gemini/refseq_nirvana_2.0.10_5bp_no_PAR_Y.bed
> This is the bed file containing RefSeq exons +/- 5bp, extracted from the Nirvana 2.0.10 Cache GRCH37 RefSeq .gff. Regions in PAR on Y have been removed. This file delineates the regions for coverage analysis in Gemini/Dias.
* :sleeping_bed: /bed_files/b37/kits/gemini/refseq_nirvana_2.0.10_100bp_no_PAR_Y.bed
> This is the bed file containing RefSeq exons +/- 100bp, extracted from the Nirvana 2.0.10 Cache GRCH37 RefSeq .gff. Regions in PAR on Y have been removed. This file delineates the regions that are variant called by Gemini/Dias.

* :sleeping_bed: /bed_files/b37/kits/gemini/regions.unique_sorted_merged_5bp.bed
> This bed file is based on Illimina Trusight Expanded clinical exome but delineates that regions that are analysed by CUH.
* :sleeping_bed: /bed_files/b37/kits/gemini/regions_tested_5bp_200521.bed
> This bed file is based on Illimina Trusight Expanded clinical exome but delineates that regions that are analysed by CUH. Updated 200521 to align with Nirvana 2.0.10 transcripts. Used with hap.py to assess variant calling performance.

* :sleeping_bed: /bed_files/b37/kits/gemini/TruSightOne.bed
> This is the bed file from Illumina containing regions captured by the TSO kit. This file is used when assessing capture wide performance (e.g. eggd_vcf_qc).
* :sleeping_bed: /bed_files/b37/kits/gemini/TSOnePlus.bed
> This is the bed file from Illumina containing regions captured by the TSO Expanded kit. This file is used when assessing capture wide performance (e.g. eggd_vcf_qc).

* :sleeping_bed: /bed_files/b37/kits/tso500/TST500C_manifest.bed
> This is the bed file provided by Illumina for Trusight Oncology 500

* :sleeping_bed: /bed_files/b38/kits/myeloid/coding_unrestricted_GRCh38_myeloid_v0.1.bed
> This is the bed file provided by TWIST and includes all exons of the genes of interest. 
* :sleeping_bed: /bed_files/b38/kits/myeloid/coding_restricted_GRCh38_myeloid_v0.1.bed
> This is the bed file provided by TWIST and includes a subset of exons of the genes of interest. 
* :sleeping_bed: /bed_files/b38/kits/myeloid/probes_GRCh38_myeo_v1.0.bed
> This is the bed file provided by TWIST and includes all probes coordinates for the myeloid panel. This file is used by eggd_picardqc_v1.0.0 

* :sleeping_bed: /bed_files/b38/kits/snp_genotyping/snp_genotyping_GRCh38_v1.0.bed
> This is the bed file provided by NimaGen and includes all SNP positions
* :sleeping_bed: /bed_files/b38/kits/snp_genotyping/snp_genotyping_amplicon_GRCh38_v1.0.bed
> This is the bed file provided by NimaGen that includes all amplicon coordinates. This file is used by eggd_picardqc_v1.0.0

* :sleeping_bed: /bed_files/b37/PAR_X.bed
> This bed file contains the PAR regions on chromosome X
* :sleeping_bed: /bed_files/b37/PAR_Y.bed
> This bed file contains the PAR regions on chromosome Y
